src/components/backend-ai-resource-preset-list.ts

```diff
@@ -325,7 +331,7 @@ class BackendAiResourcePresetList extends BackendAIPage {
           </wl-button>
         </div>
       </backend-ai-dialog>
-      <backend-ai-dialog id="delete-resource-preset-dialog" fixed backdrop blockscrolling>
+      <backend-ai-dialog id="delete-resource-preset-dialog" fixed backdrop blqockscrolling>
```

Comment:
Is attribute `blqockscrolling` exists, or just typo?

Reply:
@adrysn  It is definitely typo. haha